nixos/modules/services/web-servers/nginx/default.nix

```diff
@@ -423,7 +423,7 @@ in
         default = false;
         type = types.bool;
         description = "
-          Enable recommended proxy settings.
+          Whether to enable recommended proxy settings if a vhost does not specify the option manually.
```

Comment:
@aanderse Is this wording any better? Not sure tbh, I based it off how another option was doing it.

Reply:
Looks good 👍